platform-api/che-core-api-infrastructure-local/src/main/java/org/eclipse/che/api/local/InMemoryFactoryStore.java

```diff
@@ -36,9 +43,33 @@
  */
 @Singleton
 public class InMemoryFactoryStore implements FactoryStore {
```

Comment:
I think it should be renamed to <code>LocalFactoryStore</code> as it stores data to fs